### PR TITLE
✨ add textarea

### PR DESCRIPTION
**Pull request by YounesBessa**

### CODE REVIEW

File: packages/kitchen/src/components/Textarea/index.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  align-items: center;
+  justify-content: center;
+  outline: none;
+  transition: all 0.2s;
+  line-height: 1;
+  user-select: none;
+  border-radius: 4px;
+  padding: 7px 10px;
+  width: 790px;
+  height: 100px;
```
Comment (AntoineKM) [inline packages/kitchen/src/components/Textarea/index.tsx:43]: should be 100%

File: packages/kitchen/src/components/Textarea/index.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  outline: none;
+  transition: all 0.2s;
+  line-height: 1;
+  user-select: none;
+  border-radius: 4px;
+  padding: 7px 10px;
+  width: 790px;
```
Comment (AntoineKM) [inline packages/kitchen/src/components/Textarea/index.tsx:42]: should be 100%

File: packages/kitchen/src/components/Textarea/index.tsx (hunk 1 above)
Comment (AntoineKM) [inline packages/kitchen/src/components/Textarea/index.tsx:43]: also add a min height ?

File: packages/kitchen/src/components/Textarea/index.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  ({
+    as: Component = "textarea",
+    placeholder,
+    disabled,
+    errored,
+    defaultValue,
+    ...props
+  }: TextareaProps) => {
+    return (
+      <Component
```
Comment (AntoineKM) [inline packages/kitchen/src/components/Textarea/index.tsx:22]: should be `<textarea`

File: packages/kitchen/src/components/Textarea/index.tsx [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        placeholder={placeholder}
+        disabled={disabled}
+        errored={errored}
+        defaultValue={defaultValue}
+        {...props}
+      ></Component>
```
Comment (AntoineKM) [inline packages/kitchen/src/components/Textarea/index.tsx:28]: should be a self closing tag

**Review by AntoineKM — CHANGES_REQUESTED** [review]
- [x] Disable `resize` capability
- [ ] Uses the same colors I used for the `Input` (#125)
- [ ] Define a color for the placeholder look at `Input` for an example

**Review by jerembdn — APPROVED** [review]

File: packages/kitchen/src/components/Textarea/index.tsx [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+    );
+  }
+)<TextareaProps>`
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  outline: none;
+  transition: all 0.2s;
+  line-height: 1;
```
Comment (AntoineKM) [inline packages/kitchen/src/components/Textarea/index.tsx:35]: why ?

File: packages/kitchen/src/components/Textarea/index.tsx [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+        placeholder={placeholder}
+        disabled={disabled}
+        defaultValue={defaultValue}
+        {...props}
+      />
+    );
+  }
+)<TextareaProps>`
+  position: relative;
+  display: flex;
```
Comment (AntoineKM) [inline packages/kitchen/src/components/Textarea/index.tsx:30]: why ?

File: packages/kitchen/src/components/Textarea/index.tsx [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+  align-items: center;
```
Comment (AntoineKM) [inline packages/kitchen/src/components/Textarea/index.tsx:31]: why ?

File: packages/kitchen/src/components/Textarea/index.tsx [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+  justify-content: center;
```
Comment (AntoineKM) [inline packages/kitchen/src/components/Textarea/index.tsx:32]: why ?

File: packages/kitchen/src/components/Textarea/index.tsx [hunk 9]
```diff
@@ -0,0 +1,61 @@
...
+  outline: none;
+  transition: all 0.2s;
+  line-height: 1;
+  user-select: none;
+  resize: none;
+  border-radius: 4px;
+  padding: 7px 10px;
```
Comment (AntoineKM) [inline packages/kitchen/src/components/Textarea/index.tsx:39]: use `theme.gap`or padding i used for `Input`

File: packages/kitchen/src/components/Textarea/index.tsx [hunk 10]
```diff
@@ -0,0 +1,61 @@
...
+  width: 100%;
```
Comment (AntoineKM) [inline packages/kitchen/src/components/Textarea/index.tsx:40]: add width and height props so we can edit the width and height

**Review by AntoineKM — CHANGES_REQUESTED** [review]

**Pull request merged**